nvflare/private/fed/server/job_runner.py

```diff
@@ -290,10 +293,9 @@ def _delete_run(self, job_id, client_sites: List[str], fl_ctx: FLContext):
     def _job_complete_process(self, fl_ctx: FLContext):
         engine = fl_ctx.get_engine()
         job_manager = engine.get_component(SystemComponents.JOB_MANAGER)
-        while not self.ask_to_stop:
+        while not self.ask_to_stop and isinstance(engine.server.server_state, HotState):
```

Comment:
I don't think we need server state to be hot here since it is trying to complete jobs that are already running.

Reply:
thanks, fixed.